app/routes.py

```diff
@@ -1 +1,113 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task 
+from flask import Blueprint, jsonify, abort, make_response, request
+from sqlalchemy import asc, desc
+from datetime import datetime
+import requests 
+import os
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
```

Comment:
Notice that you have the same exact method in two files. Instead of repeating the code, it can be pulled into a helper file and imported into the two route files.